guides/gitconfig.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Managing Git Configuration
+description: Learn how to configure Git with Coder.
+---
+
+This guide will show you how to manage Git configuration through the system,
+global, and repo levels.
+
+## How to use the System and Global configurations
+
+For scenarios whereby a set of git configurations should apply across an organization,
+we recommend using a System-level git configuration. System-level git configurations
+are located at `/etc/gitconfig`. These settings are applied to each git repository,
+but may be overriden using global or worktree git configurations.
+
+We suggest adding the System-level `.gitconfig` directly in the image Dockerfile.
```

Comment:
let me know if "image Dockerfile" is correct. it sounds a bit off but I think it's the right way to say it.

Reply:
I searched to see how Docker words it and they use "image's Dockerfile"